web-server/requirements.txt

```diff
@@ -1,2 +1,3 @@
 Django
-
+django-leaflet
+python-decouple
```

Comment:
Maybe specify which version that is required.

Reply:
Fixed! Thank you :)